py/picca/cf.py

```diff
@@ -102,17 +102,21 @@ def fast_cf(z1,r1,w1,d1,z2,r2,w2,d2,ang,same_half_plate):
     w12 = w1*w2[:,None]
     z = (z1+z2[:,None])/2
 
-    w = (rp<rp_max) & (rt<rt_max) & (rp>rp_min)
+    if rp_min !=0:
+        w = (rp<rp_max) & (rt<rt_max) & (rp>rp_min)
+    else: 
+        w = (rp<rp_max) & (rt<rt_max) & (rp>=rp_min)
```

Comment:
@vserret I don't understand this if, the second case works fine for any rp_min no?

Reply:
no, if you want to have something really symmetric but I can change it